From 67b0249336fe2817f4ca82a9eba3bd7d1f076f6b Mon Sep 17 00:00:00 2001
From: Steven Toth <steven.toth@ltnglobal.com>
Date: Mon, 16 Oct 2023 15:37:25 -0400
Subject: [PATCH] spec file missing bin tool

---
 rpm/ltntstools.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ltntstools.spec b/rpm/ltntstools.spec
index 8d75488..d24daa6 100644
--- a/rpm/ltntstools.spec
+++ b/rpm/ltntstools.spec
@@ -44,6 +44,7 @@ A tool to capture, inspect or monitor MPEG-TS files and streams.
 /usr/local/bin/tstools_smpte2038_inspector
 /usr/local/bin/tstools_srt_transmit
 /usr/local/bin/tstools_ntt_inspector
+/usr/local/bin/tstools_sei_latency_inspector
 /usr/local/share/man/man8/tstools_pcapts.8
 /usr/local/share/man/man8/tstools_ffmpeg_metadata.8
 
-- 
1.8.3.1

